main/guides/getting-started/start-a-project.md

```diff
@@ -1,139 +1,292 @@
-# Starting a Project
+# Your First Agoric Dapp
+Getting your first Agoric application up and running!
```

Comment:
Dapp and application... do those mean the same thing? Are you using different words on purpose?

Is this not a term we want to define on first usage?

For example:

https://github.com/Agoric/documentation/blob/312d426b33d1f5be23d6d40fc980b2d801728390/main/guides/getting-started/README.md?plain=1#L3